src/main/java/uk/gov/hmcts/reform/sscs/scheduler/SscsCaseLoaderScheduler.java

```diff
@@ -25,7 +25,7 @@
     }
 
     @Scheduled(cron = "${sscs.case.loader.cron.schedule}")
-    public void run() {
+    void run() {
```

Comment:
Why the access defaulted to package instead of public?.

Reply:
it is highlighted by IntelliJ to follow OO best practices such as encapsulation.